MaterialDesignThemes.Wpf/DrawerOpenedEventArgs.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Windows;
+using System.Windows.Controls;
+
+namespace MaterialDesignThemes.Wpf
+{
+    public class DrawerOpenedEventArgs : RoutedEventArgs
+    {
+        public DrawerOpenedEventArgs(Dock dock, RoutedEvent routedEvent)
```

Comment:
Rather than setting RoutedEvent directly in this constructor you can pass it into the base class.
```
 public DrawerOpenedEventArgs(Dock dock, RoutedEvent routedEvent) : base(routedEvent)
```